### PR TITLE
Add spicy rock pizza

### PR DESCRIPTION
**Pull request by mkanke-real**
## About the PR

Spicy rock pizza (uranium laced pizza) has been added to the game as a new fun way to consume uranium.
## Why / Balance

Why should we have only one way to consume uranium? Burgers have so many wacky flavors, its only fair pizza gets one too! To solve the lack of wacky pizza this PR adds an item named spicy rock pizza. It is not something that would be made often as chili peppers are not a common botony product currently and uranium is generally no where near a kitchen. It deals minor radiation damage but in order to crit via eating a good deal has to be consumed, so unless a antag is force feeding it to someone so it is relatively safe.
## Technical details

This pizza flavor can be make using 1 uranium sheet, 1 flat dough, 2 chili peppers and two tomatoes for 30 seconds in the microwave. Upon consumption it will deal radiation damage via the radium reagent over time. The pizza glows using the point light component so its instantly distinct from all other pizza flavors so a character can avoid accidental consumption. Under the pizza party event it falls under the rare category so it is not something that will be seen during most gameplay sessions unless specifically sought out.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

add: Spicy rock pizza is craftable in the microwave.
Spicy Rock Pizza Example

### CODE REVIEW

**Comment by slarticodefast** [comment]
I love the idea and artwork :)
Some minor things:
- remove the empty line 575
- The values for Nutriment is way too high compared to other pizzas
- The the individual reagents should add up to the max volume. You currently have more reagents inside the pizza than its volume allows
- 1 sheet of uranium has 8 uranium and 2 radium in it (see \Resources\Prototypes\Entities\Objects\Materials\Sheets\other.yml). If you cook that into pizza it turns into 80 radium. You should try to keep the reagents the same.
grafik
- A FoodPizzaUranium has 100 Nutriment and 80 radium, but you can turn it into 8 slices with 50,10 each, meaning you have a total of 400 Nutriment and 40 Radium after cutting it. These numbers should match up.
- (looks like something similar is happening with other pizza types when slicing because the number of slices got changed from 6 to 8 at some point, but that is not your fault)

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml [hunk 1]
```diff
@@ -547,3 +547,73 @@
...
+
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 100
+        - ReagentId: Radium
+          Quantity: 80
```
Comment (LankLTE) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:584]: 35-40 total reagents is the standard, this is far too high.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml [hunk 2]
```diff
@@ -547,3 +547,73 @@
...
+    radius: 2
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 50
+        - ReagentId: Radium
+          Quantity: 10
```
Comment (LankLTE) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:617]: Ditto, this should be 5 tops.

File: Resources/Prototypes/Recipes/Cooking/meal_recipes.yml [hunk 3]
```diff
@@ -590,6 +590,16 @@
     FoodDonkpocketWarm: 3
     FoodCheeseSlice: 1
     FoodTomato: 1
+- type: microwaveMealRecipe
```
Comment (LankLTE) [inline Resources/Prototypes/Recipes/Cooking/meal_recipes.yml:593]:
```suggestion
    FoodTomato: 1
    
- type: microwaveMealRecipe
```
Should have a line after the previous recipe.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml [hunk 4]
```diff
@@ -547,3 +547,73 @@
...
+  - type: Tag
+    tags:
+    - Meat
+    - Pizza
+  - type: PointLight
+    enabled: true
+    radius: 2
+
+  - type: SolutionContainerManager
+    solutions:
```
Comment (LankLTE) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml:577]:
```suggestion
    enabled: true
    radius: 2
  - type: SolutionContainerManager
    solutions:
```
Remove the random space here.

**Review by LankLTE — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml (hunk 1 above)
Comment (mkanke-real) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml]: I went ahead and fixed this and also split the reagents between nutriment, radium and uranium to better relate to the material thats used to craft it, keep quantities consistent between cut and uncut pizza and Bringing it down to 40 reagent total for a full pizza.

**Comment by LankLTE** [comment]
Where was the sprite taken from? You should update the meta.yml's copyright field if you made it yourself.

**Comment by mkanke-real** [comment]
> Where was the sprite taken from? You should update the meta.yml's copyright field if you made it yourself.

Went ahead an added it to the very end of the other copyright info as its a sprite edit of the margherita pizza sprite. If there is a better way to add the credit in let me know!

**Review by LankLTE — APPROVED** [review]

**Pull request merged**